core/client/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -353,11 +381,14 @@ private long getCurrentBlockId() {
    * Logs IO exceptions thrown in response to the worker cache request. If the exception is not an
    * expected exception, a warning will be logged with the stack trace.
    */
-  private void handleCacheStreamIOException(IOException e) throws IOException {
+  private void handleCacheStreamIOException(IOException e) {
     if (e.getCause() instanceof BlockAlreadyExistsException) {
-      LOG.warn(BLOCK_ID_EXISTS_SO_NOT_CACHED, getCurrentBlockId());
+      // This can happen if there are two readers trying to cache the same block. The first one
+      // created the block (either as temp block or committed block). The second sees this
+      // exception.
+      LOG.info(BLOCK_ID_EXISTS_SO_NOT_CACHED, getCurrentBlockId());
     } else {
-      LOG.warn(BLOCK_ID_NOT_CACHED, getCurrentBlockId(), e);
+      LOG.info(BLOCK_ID_NOT_CACHED, getCurrentBlockId());
```

Comment:
It prints some unnecessary msgs on the spark-shell. I changed this back to LOG.warn but removed the exception in it.  I forgot the exact situation this happened. I might need to change this back to info after the test. For now, it is warn